tests/dummy/app/pods/components/user-menu/template.hbs

```diff
@@ -0,0 +1,46 @@
+{{frost-button
+  class=(concat css '-user-button')
+  hook=(concat hookPrefix '-userButton')
+  hookQualifiers=hookQualifiers
+  icon='user'
+  pack='frost-nav'
+  text=username
+}}
+
+{{#frost-popover
+  class='frost-navigation-user-menu'
```

Comment:
Never mind - this makes sense, but do you want the other CSS concats in this file?